ext/tls/lib.rs

```diff
@@ -164,12 +217,32 @@ pub fn create_http_client(
   ca_data: Option<Vec<u8>>,
   proxy: Option<Proxy>,
   unsafely_ignore_certificate_errors: Option<Vec<String>>,
+  cert_chain: Option<String>,
+  private_key: Option<String>,
```

Comment:
Let's turn this into a single option. It will let you remove some of the checks further down, and instead move them to the create http client op.

```suggestion
  client_cert_chain_and_key: Option<(String, String)>,
```